h/api/search/query.py

```diff
@@ -56,16 +56,15 @@ def build(request_params, userid=None, search_normalized_uris=False):
     if uri_param is None:
         pass
     elif search_normalized_uris:
-        filters.append(_term_clause_for_uri(uri_param))
+        filters.append(_filter_for_uri_term(uri_param))
     else:
-        matches.append(_match_clause_for_uri(uri_param))
+        filters.append(_filter_for_uri_match(uri_param))
```

Comment:
So hold on, I thought `search_normalized_uris` was a feature flag just so that we don't necessarily have to deploy the new (less tested) URI normalized search yet? But `search_normalized_uris = True` is definitely the intended behaviour on the long-term, no? i.e. I thought we'd one day remove that feature flag and remove the old `_match_clause_for_uri()` code?

But now I'm confused because here you're changing the _old_ behaviour.